src/lib.rs

```diff
@@ -9,6 +9,7 @@
 #![no_std]
 
 extern crate alloc;
+pub use libtock_codegen::main;
```

Comment:
Could you put this line under the `pub mod` section?

Reply:
Done.